src/main/java/rwg/support/SupportTC1.java

```diff
@@ -0,0 +1,37 @@
+package rwg.support;
```

Comment:
There's no need for this class because logic can be implemented in existing one. Just remove it.